pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: Automatic sourcecode generation for Dart
 homepage: https://github.com/dart-lang/source_gen
 environment:
-  sdk: '>=1.12.0 <2.0.0'
+  sdk: '>=1.23.0-dev.11.0 <2.0.0'
```

Comment:
Strictly speaking, this package doesn't *need* to align w/ the SDK version. Just the analyzer version.

One could generate code with SDK X that doesn't run in SDK X.

...right?